universe/Species.h

```diff
@@ -218,8 +218,11 @@ class FO_COMMON_API SpeciesManager {
     { bool operator()(const std::map<std::string, std::unique_ptr<Species>>::value_type& species_entry) const; };
 
 public:
-    using SpeciesTypeMap = std::map<std::string, std::unique_ptr<Species>>;
-    using iterator = SpeciesTypeMap::const_iterator;
+    using SpeciesTypeMap = std::pair<
```

Comment:
I'd have two variables, rather than making things complicated by wrapping two containers into a pair and then having `.first` and `.second` everywhere...

Reply:
I will try to change that.

Reply:
Fixed with e59ea1e066896434c8b4b3c6da1b79d409219edd